release_notes/Service_Fabric_ReleaseNotes_81.md

```diff
@@ -0,0 +1,111 @@
+| **Windows - 8.1.316.9590<br>Ubuntu 16 - 8.1.320.1<br>Ubuntu 18 - 8.1.320.1** | **Feature** | **Preview** Add support for .NET 6.0 Service Fabric applications | **Brief desc**: Service Fabric version 8.1 introduces programming model support for .NET 6.0 Service Fabric applications.<br> |
+| **Windows - 8.1.316.9590<br>Ubuntu 16 - 8.1.320.1<br>Ubuntu 18 - 8.1.320.1** | **Feature** | Add API support application description update | **Brief desc**: Service Fabric version 8.1 introduces REST API support for updating Service Fabric applications. Customer can specify application name, which is a required parameter, flags, which denote which of the parameters will be updated, maximum nodes, minimum nodes, application metrics and whether application capacities should be removed.<br>Corresponding PowerShell API is documented here: https://docs.microsoft.com/en-us/powershell/module/servicefabric/update-servicefabricapplication?view=azureservicefabricps.<br>**Usage**: Added UpdateApplication REST API<br>**Impact**: Customer is able to update application with REST API.<br> |
+| **Windows - 8.1.316.9590<br>Ubuntu 16 - 8.1.320.1<br>Ubuntu 18 - 8.1.320.1** | **Feature** | Add periodic ping between RA and RAP to detect IPC failure and process stuck | **Brief desc**: Add periodic ping between RA and RAP to detect IPC failure and process stuck<br>**Usage**: On every node, the Service Fabric agent on the node will periodically exchange a heartbeat message with the customer process, to ensure the IPC is working and the process is not frozen. When such issues have been detected (not a single heartbeat message has been successfully exchanged in 15 min by default), the process will be terminated to mitigate the issue.<br>**Impact**: Add periodic ping between RA and RAP to detect IPC failure and process stuck<br> |
+| **Windows - 8.1.316.9590<br>Ubuntu 16 - 8.1.320.1<br>Ubuntu 18 - 8.1.320.1** | **Feature** | Add support Probes for non containerized applications | **Brief desc**: Support for liveness and readiness probe for non containerized applications<br> |
+| **Windows - 8.1.316.9590<br>Ubuntu 16 - 8.1.320.1<br>Ubuntu 18 - 8.1.320.1** | **Feature** | Make cluster upgrade for node capacity updates impactless | **Brief desc**: Service Fabric version 8.1 introduces support for dynamic upgrade of node capacity and node property configs, with no disruption to running workloads.
```

Comment:
Do we have any description of what the previous behavior was?

Reply:
for which one?

Reply:
Periodic ping I think he's referring to, this is a new feature so it would just freeze and would have to be mitigated by engineers.

"the process" < which process? IPC or customer process?

Reply:
Its the customer process. Updated accordingly.